File: app-marketplace/marketplace-frontend-fix/src/app/models/DTO/UserEditDTO.ts
import { Establishment } from "../Entities/Establishment";

export interface UserEditDTO{
    id :number,
    name :string,
    password :string,
    phoneNumber :string,
    profileId :number,
    email :string,
    documentNumber :string,
    birthdayDate :string

    userEstablishments :Establishment[];
}

File: app-marketplace/marketplace-backend/Backend.Domain/DTO/UserEditDTO.cs
using Backend.Domain.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Backend.Domain.DTO
{
    public class UserEditDTO
    {
        public long Id { get; set; }
        public required string Name { get; set; }
        public required string Password { get; set; }
        public required string PhoneNumber { get; set; }
        public int ProfileId { get; set; }
        public required string BirthdayDate { get; set; }

        public List<Establishment>? UserEstablishments { get; set; }
        
    }
}
